common/src/main/java/org/apache/drill/common/config/DrillConfig.java

```diff
@@ -138,34 +154,83 @@ public static DrillConfig create(Properties testConfigurations) {
     return create(null, testConfigurations, true);
   }
 
-  public static DrillConfig create(String overrideFileName, boolean enableServerConfigs) {
-    return create(overrideFileName, null, enableServerConfigs);
+  /**
+   * ...
+   * @param overrideFileResourcePathname
+   *          see {@link #create(String)}'s {@code overrideFileResourcePathname}
+   */
+  public static DrillConfig create(String overrideFileResourcePathname, boolean enableServerConfigs) {
+    return create(overrideFileResourcePathname, null, enableServerConfigs);
   }
 
-  private static DrillConfig create(String overrideFileName, Properties overriderProps, boolean enableServerConfigs) {
-    overrideFileName = overrideFileName == null ? CommonConstants.CONFIG_OVERRIDE : overrideFileName;
-
-    // first we load defaults.
+  /**
+   * ...
```

Comment:
I haven't seen this convention before, but I assume this means that the docs for similar methods suffice and don't need to be repeated here. I would possibly recommend moving the longer comment above to here, in which case I think it would be possible to just put "..." above all of the alternative implementations of this method. I think it makes the most sense to put the comment describing the general idea of the functions purpose above the version that also describes what all of parameters can do.